### DISCUSSION TITLE
Error 401 with 'getReviewsByShop' and 'getReviewsByListing' even on my own shop

### DISCUSSION THREAD

**Discussion by RodiCoci** [etsy/open-api#1226] [Q&A]
Hello, 

When I'm trying to get the reviews of a shop through 'getReviewsByShop' or 'getReviewsByListing', I get error 401 (The request lacks authentication credentials). But my credentials (only api-key) are valid and work on other API requests (e.g. 'getShop').

Is this normal? Am I missing something? Does somebody knows what I'm doing wrong?

The code I'm using is:

def Functie_Get_Shop_Reviews(shop_id, api_key, limit=25, offset=0, min_created=None, max_created=None):
    
    url = f"https://api.etsy.com/v3/application/shops/{shop_id}/reviews"
    
    params = {
        'api_key': api_key,
        'limit': limit,
        'offset': offset,
        'min_created': min_created,
        'max_created': max_created
    }
    
    response = requests.get(url, params=params)
    
    if response.status_code == 200:
        # Als succesvol, parse en retourneer de JSON data
        return response.json()
    else:
        # Anders, retourneer een error bericht
        return {'error': 'Failed to fetch reviews', 'status_code': response.status_code}

**Comment by Gareth064** [comment]
API key doesn't go into the params, its meant go as a header
